utils/src/field_helpers.rs

```diff
@@ -1,7 +1,7 @@
-use ark_ff::FftField;
+use ark_ff::Field;
 
 /// Field element helpers
-pub trait FieldHelpers<F: FftField> {
+pub trait FieldHelpers<F: Field> {
```

Comment:
Perhaps here use `FieldHelpers<F>` , without specifying the concrete generic type, and have it only in the implementation.